src/llmcompressor/observers/base.py

```diff
@@ -73,11 +79,14 @@ def post_calculate_qparams(self) -> None:
         Run any logic specific to its observers after running calculate_qparams
         """
 
+    # TODO: use a different name?
```

Comment:
rather than having the return signature of `get_qparams` change based on these bool values, what if we just had two separate methods `def get_gparams` and `def get_qparams`?

Reply:
yeah I refactor it out a bit to clean it up